Examples/Tests/Langmuir/inputs_3d_rt

```diff
@@ -56,5 +56,5 @@ electrons.momentum_distribution_type = "constant"
 
 # Diagnostics
 diagnostics.diags_names = diag1
-diag1.period = 100
+diag1.period = 40
```

Comment:
I changed it from 100 to 40 when I was trying to fix the failing test, since for this test we have `max_step=40` and it seemed more consistent to have 'diag1.period=40'. Then I figured that first and last iterations are written anyway, so it did not matter, but I forgot to revert it back. Thanks for noticing, reverted!

Reply:
Thank you @oshapoval for explaining. I did not notice that there was an inconsistency (even if harmless) between `max_step` and `diag1.period` in this input file. Thank you for noticing this, we can keep in mind to clean it up at some point.